### ISSUE TITLE
neovim not able to find syntax.vim under windows

### ISSUE THREAD

**Issue by RedX2501** [neovim/neovim#3852]
I'm getting the following message:

```
D:\Playground\neovim\neovim-qt>d:\Playground\neovim\Neovim\bin\nvim.exe -V9log.txt
Error detected while processing C:\Users\uidw8099\AppData\Roaming\nvim\init.vim:
line   90:
E484: Can't open file d:\Playground\neovim\Neovim\bin\syntax\syntax.vim
Press ENTER or type command to continue^C
```

I have `set XDG_CONFIG_HOME=%APPDATA%\` before starting neovim.exe in a windows command prompt.

I don't have any settings related to syntax.vim expect `syntax on` in my `init.vim`.

I have created `%APPDATA%\nvim\syntax\syntax.vim` but it did not help and I have found that `syntax.vim` (as expected) is shipped with neovim.

I don't get why it's looking under the `bin` dir but if i create it, neovim will start.

I think this might be a lookup bug for windows.

**Comment by equalsraf** [comment]
Does setting the environment variable VIMRUNTIME to point to the runtime/ folder help?

In Windows Vim used to look for the runtime folder in the same folder as the running binary, #810 used to have changes to the install targets to address this, but they no longer work and the runtime folder is being placed in share/nvim while the binary is in bin/nvim.exe. So we either change the install rules or the runtime path lookup.

**Comment by RedX2501** [comment]
@equalsraf Yes it helps. Now i even get a message about missing clipboard provider. Without settin that i never got the message.

**Comment by equalsraf** [comment]
I've pushed a fix for this into #810, it should no longer be necessary to set $VIMRUNTIME.

**Comment by justinmk** [comment]
Doesn't appear to be an issue anymore.
